### PR TITLE
Updated maintainers

### PR DESCRIPTION
**Pull request by NikCharlebois**
Updated maintainers for OfficeOnlineServerDSC and SharePointDSC

---
This change is [Reviewable](https://reviewable.io/reviews/powershell/dscresources/352)

### CODE REVIEW

**Comment by johlju** [comment]
:lgtm:

@kwirkykat can you please look at this PR updating maintainers in SharePointDsc and OfficeOnlineServerDsc and merge if it looks okay! 

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/powershell/dscresources/352#-:-LAM_2s8CqVsOwC52TyS:b-4copr9)*

File: Maintainers.md [hunk 1]
```diff
@@ -32,10 +32,10 @@ All repositories without dedicated maintainers are maintained by the current DSC
 | AuditPolicyDsc | Adam Haynes ([@athaynes](https://github.com/athaynes)) |
 | DscResources | Katie Keim ([@kwirkykat](https://github.com/kwirkykat)) <br/> Zachary Alexander ([@zjalexander](https://github.com/zjalexander)) |
 | DscResource.Tests | Katie Keim ([@kwirkykat](https://github.com/kwirkykat)) <br/> Mariah Breakey ([@mbreakey3](https://github.com/mbreakey3)) |
-| OfficeOnlineServerDsc | Brian Farnhill ([@BrianFarnhill](https://github.com/BrianFarnhill)) <br/> Nik Charlebois ([@NikCharlebois](https://github.com/NikCharlebois)) |
+| OfficeOnlineServerDsc | Nik Charlebois ([@NikCharlebois](https://github.com/NikCharlebois)) |
 | PSDscResources | Mariah Breakey ([@mbreakey3](https://github.com/mbreakey3)) <br/> Katie Keim ([@kwirkykat](https://github.com/kwirkykat)) |
 | SecurityPolicyDsc | Jason Walker ([@jcwalker](https://github.com/jcwalker)) |
-| SharePointDsc | Brian Farnhill ([@BrianFarnhill](https://github.com/BrianFarnhill)) |
+| SharePointDsc | Yorick Kuijs([@YorickKuijs](https://github.com/YKuijs))<br/> Nik Charlebois ([@NikCharlebois](https://github.com/NikCharlebois)) |
```
Comment (kwirkykat) [inline Maintainers.md]: Looks like Yorick's tag is actually ykuijs instead of YorickKuijs

**Comment by johlju** [comment]
@NikCharlebois Do you have time to resolve the typo?

**Comment by NikCharlebois** [comment]
@johlju Done

**Comment by johlju** [comment]
@NikCharlebois Thanks for that. There are merge conflicts, can you resolve those when you got a chance? 🙂

**Comment by johlju** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[Maintainers.md, line 38 at r2](https://reviewable.io/reviews/powershell/dscresources/352#-LC2WBNt37wMkO-mk0NJ:-LC2WBNt37wMkO-mk0NK:be8djnb) ([raw file](https://github.com/powershell/dscresources/blob/9ac9fdf143a32751cded119447136733b87af2c6/Maintainers.md#L38)):*
> ```
> Yorick Kuijs(
> ```

Seems that a space is missing before the parenthesis.

---


*Comments from [Reviewable](https://reviewable.io/reviews/powershell/dscresources/352)*

**Comment by johlju** [comment]
@NikCharlebois Just a tiny space that is needed, then I can merge this. :)

**Comment by johlju** [comment]
:lgtm:

I will merge this and send another PR to fix the last review comment.

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io/reviews/powershell/dscresources/352#-:-LDBR8HTE7rCZJy_EUoJ:bz9wh52)*

**Pull request merged**